model/bootstrap/Oauth2OnlySessionBuilder.php

```diff
@@ -18,16 +18,19 @@
  *
  */
 
-namespace oat\taoOauth\model;
+namespace oat\taoOauth\model\bootstrap;
 
-/**
- * Interface OauthController
- *
- * To identify a controller that require Oauth identification
- *
- * @package oat\taoOauth\model
- */
-interface OauthController
+class Oauth2OnlySessionBuilder extends Oauth2SessionBuilder
```

Comment:
This entire class looks like a configuration of `Oauth2SessionBuilder`, a config like 'oauth_only'  but this means having you will have to make it a ConfigurableService or a config of extension.